src/html/template-landing.html

```diff
@@ -0,0 +1,246 @@
+            <a class="usa-nav-link" href="#">
+              <span>Distinct from each other</span>
+            </a>
+          </li>
+          <li>
+            <button class="usa-accordion-button usa-nav-link" aria-expanded="false" aria-controls="sidenav-3">
+              <span>More details in sub-menus</span>
+            </button>
+            <ul id="sidenav-3" class="usa-nav-submenu">
+              <li>
+                <a href="#">Page title</a>
+              </li>
+              <li>
+                <a href="#">Page title</a>
+              </li>
+              <li>
+                <a href="#">Page title</a>
+              </li>
+            </ul>
+          </li>
+        </ul>
+        <div class="usa-nav-secondary">
+          <form class="usa-search usa-search-small js-search-form">
+            <div role="search">
+              <label class="usa-sr-only" for="search-field-small">Search small</label>
+              <input id="search-field-small" type="search" name="search">
+              <button type="submit">
+                <span class="usa-sr-only">Search</span>
+              </button>
+            </div>
+          </form>
+          <ul class="usa-unstyled-list usa-nav-secondary-links">
+            <li class="js-search-button-container">
+              <button class="usa-header-search-button js-search-button">Search</button>
+            </li>
+            <li>
+              <a href="#">Secondary priority link</a>
+            </li>
+            <li>
+              <a href="#">Easy to comprehend</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+  </header>
+  <div class="usa-overlay"></div>
+  <main id="main-content">
+    <section class="usa-hero">
+      <div class="usa-grid">
+        <div class="usa-hero-callout usa-section-dark">
+          <h2><span class="usa-hero-callout-alt">Hero callout:</span> Call attention to a current priority</h2>
+          <a class="usa-hero-link" href="#">Link to more about that priority</a>
+          <a class="usa-button usa-button-big usa-button-secondary" href="#">Learn about what we do</a>
+        </div>
+      </div>
+    </section>
+    <section class="usa-grid usa-section">
+      <div class="usa-width-one-third">
+        <h2>A tagline highlights your approach.</h2>
+      </div>
+      <div class="usa-width-two-thirds">
+        <p>The tagline should inspire confidence and interest, focusing on the value that your overall approach offers to your audience. Use a heading typeface and keep your tagline to just a few words, and don’t confuse or mystify.</p>
+        <p>Use the right side of the grid to explain the tagline a bit more. What are your goals? How do you do your work? Write in the present tense, and stay brief here. People who are interested can find details on internal pages.</p>
+      </div>
+    </section>
+    <section class="usa-section usa-section-dark usa-graphic_list">
+      <div class="usa-grid usa-graphic_list-item">
```

Comment:
I see what you mean. The `usa-graphic_list-item` is a row of media blocks. Happy to change it if you think "row" is clearer than "item." I abstractly use "item" sometimes to refer to a single instance of "a thing" and see how it can be confusing.